### DISCUSSION TITLE
input.insert_mode.auto_enter not working on Trello.com

### DISCUSSION THREAD

**Discussion by cmvanb** [qutebrowser/qutebrowser#7350] [Q&A] [answered]
Normally insert mode is activated whenever an input element is active, so it becomes second nature to simply type. Unfortunately it doesn't work for me on Trello.com.

Reproduce steps:

1. Navigate to Trello.com
2. Create an account if necessary.
3. Open an existing board, or create one.
4. Create a list or a card in a list- using either the mouse or the key hint mechanism.
5. Observe the blinking cursor, indicating an active input element.
6. Observe insert mode is not active.

This means I need to  perform every action twice (key hinting or mouse clicking) in order to insert text.

Is there any hack/workaround I can use to make this website work? Or am I better off logging an issue?

best regards, love this browser

**Comment by interdisziplina** [comment]
You can enter insert mode manually simply by pressing `i`, there is no need to click / focus the element again.

**Reply by cmvanb** [reply]
Yes- but that is not the desired behavior. If the cursor is blinking it should be in insert mode already.

**Comment by The-Compiler** [comment, marked as answer]
It's a well-known issue, but unfortunately a bit tricky to fix. Relevant issues:

- #2668
- #2471

**Reply by cmvanb** [reply]
I see, thank you for highlighting the relevant issues, good to know it is being tracked.
